airflow/providers/qubole/operators/qubole.py

```diff
@@ -267,7 +267,7 @@ def get_hook(self) -> QuboleHook:
         return QuboleHook(**self.kwargs)
 
     def __getattribute__(self, name: str) -> str:
-        if name in QuboleOperator.template_fields:
+        if name in _get_template_fields(self):
```

Comment:
what is the rationale behind this change?

Reply:
It avoids hardcoding the class name `QuboleOperator` here, so you can subclass it safely. i.e. you can see it in `qubole_check.py`